.github/workflows/test.yml

```diff
@@ -11,6 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      
+      - name: Set Node.js 16.x
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: 16.x
```

Comment:
ooc, what's this about?

Reply:
We were getting an error in CI because the workflow was using a Node version > 16. So, I just specified the version as `16.x` (we already do this in the other workflow). I spoke with Suraj about this earlier today.

Reply:
roger